ephemeraldata.adoc

```diff
@@ -35,40 +35,85 @@ through the following process:
    <<_ssdb_configuration_options>> for more information.
-2. Configure the {product-name} to utilize `SSDB`.
-3. Migrate any existing short-lived data to `SSDB` using the `/opt/zimbra/bin/zmmigrateattrs` command.
+2. Migrate any existing short-lived data to `SSDB` using the `/opt/zimbra/bin/zmmigrateattrs` command.
+3. Configure the {product-name} to utilize `SSDB`.
+
 
 == Migration Procedure
 
 1. Access the command prompt on one of the machines in the installation.
-2. Configure the {product-name} to use `SSDB`:
 
-You may use either an ip address or a hostname for the host portion of the
-configuration item.  In this case you will need to ensure it resolves to the
-proper ip address on all machines in the cluster. If the value does not resolve to a functioning
-backend, the value of `zimbraEphemeralBackendURL` will not be changed.
+[start=2]
+. Migrate existing ephemeral data to the `SSDB` backend using the `zmmigrateattrs` utility
 
 ----
 sudo su - zimbra
-zmprov mcf zimbraEphemeralBackendURL ssdb:<ip address|hostname>:port # substituting your server values
+/opt/zimbra/bin/zmmigrateattrs ssdb:<ip address|hostname>:port # substituting your server values
 ----
 
+You may use either an ip address or a hostname for the host portion of the destination URL.
+Either way, you will need to ensure it resolves to the proper ip address on all machines in the cluster.
+If the provided SSDB address does not resolve to a functioning backend, the migration process will terminate.
+
+
 [start=3]
-. Migrate any existing attributes from LDAP -> SSDB
+. Configure {product-name} to use `SSDB`:
 
 ----
 sudo su - zimbra
-/opt/zimbra/bin/zmmigrateattrs -n 4
+zmprov mcf zimbraEphemeralBackendURL ssdb:<ip address|hostname>:port # substituting your server values
 ----
 
-== Additional Notes on Ephemeral Data
+As with migration, the host and port must resolve to a functioning SSDB backend. Otherwise,
+the value of `zimbraEphemeralBackendURL` will not be changed.
+
+
+== Migration Details
+
+=== Migration Info
+
+Information about the latest migration process can be viewed by running the command `zmmigrateattrs --status`.
+If the migration is currently in progress, this command may have to be run from a new terminal window.
+This command will output three pieces of information:
+
+1. The status of the migration: one of IN_PROGRESS, COMPLETED, or FAILED
+2. The URL of the SSDB backend acting as the destination
+3. A timestamp of when the migration process was initiated
+
+The migration info can be reset with the command `zmmigrateattrs --clear`. This should only be done if
+the status does not reflect the true state of the system.
+
+=== Changing the Ephemeral Backend URL
+
+When the value of zimbraEphemeralBackendURL is modified, the {product-name} checks the status of the last known migration.
+This can result in one of several scenarios:
+
+1. If the migration is completed and the URL of this migration matches the newly provided value, zimbraEphemeralBackendURL
```

Comment:
Looks good overall.  I would add some markup to things like the names of ldap attributes so they are rendered in monospace; e.g., `zimbraEphemeralBackendURL`, as is done elsewhere.